lib/Controller/BoardController.php

```diff
@@ -155,4 +155,15 @@ public function deleteAcl($aclId) {
 	public function clone($boardId) {
 		return $this->boardService->clone($boardId, $this->userId);
 	}
+
+	/**
+	 * @NoAdminRequired
+	 * @param $boardId
+	 * @param $owner
+	 * @param $newOwner
+	 * * @return null|void
+	 */
+	public function transferOwner($boardId, $owner, $newOwner) {
```

Comment:
```suggestion
	public function transferOwner(int $boardId, string $newOwner): DataResponse {
```

Things to add here:
- a permission check if the current user is actually the owner
- then the owner does not need to be passed as a parameter anymore through the API